clients/python/test/e2e/test_notebooks.py

```diff
@@ -45,3 +46,90 @@ def test_run_notebooks(tmp_path: Path, notebook: Path, params: dict[str, Any] =
         kernel_name="python3",
         parameters=params,
     )
+
+
+def get_tutorials(osparc_version: Optional[str] = None) -> List[Path]:
+    """Returns the tutorial notebooks compatible with a given osparc client version
+
+    Args:
+        osparc_version (str): osparc.__version__
+
+    Returns:
+        List[Path]: A list of *Path*s to the tutorial notebooks
+    """
+    compatibility_dict: Dict[str, Any] = json.loads(
+        TUTORIAL_CLIENT_COMPATIBILITY_JSON.read_text()
+    )
+    tutorial_names: Set[str] = set()
+    if osparc_version is not None:
+        assert (
+            osparc_version in compatibility_dict["versions"]
+        ), f"{osparc_version} does not exist in {TUTORIAL_CLIENT_COMPATIBILITY_JSON}"
+        tutorial_names = set(compatibility_dict["versions"][osparc_version])
+    else:
+        for v in compatibility_dict["versions"]:
+            tutorial_names = tutorial_names.union(
+                set(compatibility_dict["versions"][v])
+            )
+    result: List[Path] = []
+    for name in tutorial_names:
+        result += list(DOCS_DIR.rglob(f"*{name}"))
+
+    return result
+
+
+# Tests -------------------------------------------------------------------------------
+
+
+def test_notebook_config(tmp_path: Path):
+    """Test configuration of test setup.
+    Make sanity checks (ensure all files are discovered, correct installations are on path etc)
+
+    Args:
+        tmp_path (Path): Temporary path pytest fixture
+    """
+    # sanity check configuration of jupyter environment
+    config_test_nb: Path = DATA_DIR / "config_test.ipynb"
+    assert config_test_nb.is_file()
+    run_notebook(
+        tmp_path,
+        config_test_nb,
+        {
+            "expected_python_bin": sys.executable,
+            "expected_osparc_version": str(osparc.__version__),
+            "expected_osparc_file": osparc.__file__,
+        },
+    )
+
+    # sanity check paths and jsons: are we collecting all notebooks?
+    tutorials: Set[Path] = set(DOCS_DIR.glob("*.ipynb"))
+    json_notebooks: Set[Path] = set(get_tutorials())
+    assert len(tutorials) > 0, f"Did not find any tutorial notebooks in {DOCS_DIR}"
+    assert (
+        len(tutorials.difference(json_notebooks)) == 0
+    ), f"Some tutorial notebooks are not present in {TUTORIAL_CLIENT_COMPATIBILITY_JSON}"
+
+    # check that version of this repo is present in compatibility json
+    cur_version: str = json.loads((API_DIR / "config.json").read_text())["python"][
+        "version"
+    ]
+    assert (
+        cur_version
+        in json.loads(TUTORIAL_CLIENT_COMPATIBILITY_JSON.read_text())["versions"].keys()
+    ), f"The version defined in {API_DIR/'config.json'} is not present in {TUTORIAL_CLIENT_COMPATIBILITY_JSON}"
+
+
+@pytest.mark.parametrize("tutorial", get_tutorials())
```

Comment:
TIP: using `ids=` argument will display the parametrization much clearer. 

e.g. try running it now and see the output vs
```python
@pytest.mark.parametrize("tutorial", get_tutorials(), ids: lambda p: p.name)
def test_...
``